### DISCUSSION TITLE
Should the role run with become?

### DISCUSSION THREAD

**Discussion by gregerspoulsen** [kyl191/ansible-role-openvpn#165] [Q&A] [answered]
Hi,

I tried to run the example playbook but it failed with:

"'/usr/bin/apt-mark manual openvpn' failed: E: Could not create temporary file for /var/lib/apt/extended_states - mkstemp (13: Permission denied)

I managed to get it running successfully with become: True.

Is that the right way to use it?

Best regards,
Gregers

**Comment by kyl191** [comment, marked as answer]
Yes, the role should be run with `become: true`
